src/swap/types.ts

```diff
@@ -102,6 +116,7 @@ export type SwapAmountInputReact = {
   swappableTokens?: Token[]; // Swappable tokens
   token?: Token; // Selected token
   type: 'to' | 'from'; // Identifies if component is for toToken or fromToken
+  className?: string; // Optional className override for top div element.
```

Comment:
are we overriding `QuoteWarning`?

Reply:
No. Will update now!

Reply:
Overriding here https://github.com/coinbase/onchainkit/pull/719/files#diff-9bca97e90be8d6cfaa2c6813dbf1d684ed792b80b395f6b033d3d06963f23371R99

Reply:
Are we sure we want to add className to QuoteWarning and RawTransactionData?

Reply:
Hmm I don't think QuoteWarning and RawTransactionData overriding is very important. I don't think we need it unless users start to ask for it.

Reply:
derp the code folding... see it now lol

We don't need `className` override to `QuoteWarning` right?

Reply:
Nope! Removed